hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewModal.tsx

```diff
@@ -180,7 +208,20 @@ const OrgUnitTreeviewModal = ({
             resetSelection();
         }
     }, [resetTrigger, hardReset, resetSelection]);
+
+    useEffect(() => {
+        if (sourceSettings && !isFetchingSource) {
+            setSettings({
+                ...settings,
+                statusSettings: sourceSettings,
+            });
```

Comment:
Wouldn't using a function solve the exhaustive deps problem?

```suggestion
            setSettings(value => ({...value, statusSettings: sourceSettings}));
```